src/Sylius/Behat/Page/Admin/Product/UpdateSimpleProductPage.php

```diff
@@ -388,26 +352,21 @@ public function getPriceForChannel($channelName)
     /**
      * {@inheritdoc}
      */
-    public function getValidationMessageForImageAtPosition($position)
+    protected function getCodeElement()
```

Comment:
Product still has a code, and uses the `CodeSmthTrait`. Since this method is abstract in that trait, then yes, we do need it.